test/GoodGhosting.test.js

```diff
@@ -2,6 +2,10 @@
 const IERC20 = artifacts.require("IERC20");
 const ERC20Mintable = artifacts.require("MockERC20Mintable");
 const GoodGhosting = artifacts.require("GoodGhosting");
+const GoodGhostingPolygon = artifacts.require("GoodGhostingPolygon");
+
+const IncentiveControllerMock = artifacts.require("IncentiveControllerMock");
+
```

Comment:
It'd be better to create a separate file `GoodGhostingPolygon.test.js`.
Since it inherits from GoodGhosting, it doesn't need to test everything again, but just the methods that are overwriten.

By doing so we'll better more modular tests, and probably we don't need the `TYPE` env variable anymore.

Let me know your thoughts. Thanks

Reply:
okay I actually did the change to avoid having another file but yeah okay also the test cases for the overwritten functions too would be the same only  there was no major change the matic checks are already covered in the polygon fork scenario's